src/components/MultiStepIndicator/components/StepIndicator/StepIndicator.module.scss

```diff
@@ -36,7 +36,7 @@
 }
 
 .textContainerTitleText {
-  font: var(--font-h5);
+  font: var(--font-text-bold);
   font-size: 14px;
```

Comment:
1. I'd expect the codemod to remove all font-related properties
2. we might want to change the codemod to replace the variables by extending the global classes

Reply:
changed it to be as we discussed before